cmd/gaia/app/app.go

```diff
@@ -73,7 +73,7 @@ func NewGaiaApp(logger log.Logger, db dbm.DB) *GaiaApp {
 	)
 
 	// add handlers
-	app.coinKeeper = bank.NewKeeper(app.accountMapper)
+	app.coinKeeper = bank.NewKeeper(app.accountMapper, nil)
```

Comment:
when we're going to be passing in `nil` most of the time, it usually makes sense to have a second function. So we would keep NewKeeper the same and add `KeeperWithBanker` that takes a keeper and returns a new one with the banker set. Default no banker, and we don't have a vague `nil` everywhere :)